src/hacbs/components/ImportForm/create-utils.ts

```diff
@@ -36,6 +37,7 @@ export const createIntegrationTest = (
     metadata: {
       name,
       namespace,
+      ...(optional && { labels: { [IntegrationTestLabels.OPTIONAL]: optional.toString() } }),
```

Comment:
just out of curiosity...is it the only label an integrationTestScenario would carry?